github/status_test.go

```diff
@@ -66,21 +65,23 @@ func TestGitHubStatusFailureMockAPI(t *testing.T) {
 		wantStatus int
 	}{
 		{
-			name:       "500 Internal Server Error",
-			body:       "Something bad happened!",
-			wantErr:    http.StatusText(http.StatusInternalServerError),
+			name: "500 Internal Server Error",
+			body: "fake body",
```

Comment:
Here "fake body" is incorrectly not reported.
The body should always be reported in case of error because it may contain additional troubleshooting help. Subsequent PR will fix this